pkg/controller/support/credentials/cloudauth/cloudauth_test.go

```diff
@@ -14,20 +19,27 @@ func TestRegistry_RetrieveAuthorization(t *testing.T) {
 		Password: "test-pass",
 	}
 	registry := &Registry{}
-	registry.Register(regexp.MustCompile(`^my.cloud`), func(ctx context.Context, url string) (*types.AuthConfig, error) {
+	registry.Register(regexp.MustCompile(`^my.cloud`), func(ctx context.Context, logger logr.Logger, url string) (*types.AuthConfig, error) {
 		return expected, nil
 	})
+	zapLogger, err := logger.NewZap(cfg.Logging{})
+	if err != nil {
+		t.Fatalf("Unexpected error while creating test logger.")
+	}
+
+	ctrl.SetLogger(zapr.NewLogger(zapLogger))
```

Comment:
I didn't see a test logger set up, I set it up like we set up the regular logger